api/python/notebooks/experimental/pytorch.ipynb

```diff
@@ -0,0 +1,675 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "9c8899e7",
+   "metadata": {},
+   "source": [
+    "## Training a PyTorch Model\n",
+    "\n",
+    "This tutorial shows how to train a Logistic Regression model in PyTorch using the Census API's `experimental.ml.ExperimentDataPipe` class. This is intended only to demonstrate the use of the `ExperimentDataPipe`, and not as an example of how to train a biologically useful model.\n",
+    "\n",
+    "This tutorial assumes a basic familiarity with PyTorch and the Census API. See the [`Querying and fetching the single-cell data and cell/gene metadata`](../api_demo/census_query_extract.ipynb) notebook tutorial for a quick primer on Census API usage.\n",
+    "\n",
+    "**Contents**\n",
+    "* [Open the Census](#Open-the-Census)\n",
+    "* [Create a DataLoader](#Create-a-DataLoader)\n",
+    "* [Define the model](#Define-the-model)\n",
+    "* [Train the model](#Train-the-model)\n",
+    "* [Make predictions with the model](#Make-predictions-with-the-model)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f874fb88",
+   "metadata": {},
+   "source": [
+    "## Open the Census\n",
+    "\n",
+    "First, obtain a handle to the Census data, in the usual manner:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "c3dd549f",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "The \"stable\" release is currently 2023-05-15. Specify 'census_version=\"2023-05-15\"' in future calls to open_soma() to ensure data consistency.\n"
+     ]
+    }
+   ],
+   "source": [
+    "import cellxgene_census\n",
+    "\n",
+    "census = cellxgene_census.open_soma()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "580b29f2",
+   "metadata": {},
+   "source": [
+    "## Create a DataLoader\n",
+    "\n",
+    "To train a model in PyTorch using this `census` data object, first instantiate an `ExperimentDataPipe` as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "54896e6f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import cellxgene_census.experimental.ml as census_ml\n",
+    "import tiledbsoma as soma\n",
+    "\n",
+    "experiment_datapipe = census_ml.ExperimentDataPipe(\n",
```

Comment:
I think this code would be clearer - given the prose - if it looked like:
```
   experiment = census["census_data"]["homo_sapiens"]
   experiment_datapipe = census_ml.ExperimentDataPipe(
        experiment,
        ...
    )
```

Reply:
Agreed, that's clearer! Fixed.